src/pages/FindYourCommunity/YourLocalArea/FindCharity/FindCharity.module.scss

```diff
@@ -15,3 +15,12 @@
   padding: 0;
   font-size: 14px;
 }
+
+@media (max-width: 700px) {
+  .container {
+    padding: 48px 20px;
+  }
+  .subContainer {
+    padding: 48px 20px;
+  }
+}
```

Comment:
Added this media query as I noticed the page looked pretty wack when viewed on a mobile device. For reference, here is what the page would've previously looked like on an iPhone 14 Pro Max:

![image](https://github.com/CapgeminiInventUK/donate-to-educate-frontend/assets/10779091/a83b10aa-b3de-404d-a0e0-e330dcf956f1)

With this media query, the same page looks like this:

![image](https://github.com/CapgeminiInventUK/donate-to-educate-frontend/assets/10779091/18c0ba4b-038f-4689-8f39-ba207b45d7ff)